pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+						tally,
+						timeline: proposalTimeline,
+						title: data?.title || subsquareTitle || null,
+						topic: topic
+							? topic
+							: isTopicIdValid(topic_id)
+							? {
+									id: topic_id,
+									name: getTopicNameFromTopicId(topic_id)
+							  }
+							: topicFromType,
+						track_no: !isNaN(trackNumber) ? trackNumber : null,
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)